.github/workflows/build.yml

```diff
@@ -0,0 +1,48 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
```

Comment:
This allows us to also trigger this workflow (i.e. a build) manually via the `Actions` tab

Reply:
Just want to check if I understand correctly, when we push a commit to a feature branch, we have to manually go to the `Actions` tab to initiate a build?

Reply:
Yes - I think that's right, although we're thinking about workflows on [another](https://github.com/guardian/mobile-fastly-cache-purger/pull/1) project and it looks like we could also create draft PRs to trigger the build too (and committing to a feature branch that has an open draft PR also triggers a build as well)...